sdk/storage/azure-storage-file/azure/storage/file/aio/directory_client_async.py

```diff
@@ -262,50 +261,61 @@ def list_handles(self, recursive=False, **kwargs):
             page_iterator_class=HandlesPaged)
 
     @distributed_trace_async
-    async def close_handles(
-            self, handle=None, # type: Union[str, HandleItem]
-            recursive=False,  # type: bool
-            **kwargs # type: Any
-        ):
-        # type: (...) -> int
-        """Close open file handles.
+    async def close_handle(self, handle, **kwargs):
+        # type: (Union[str, HandleItem], Any) -> int
+        """Close an open file handle.
 
         :param handle:
-            Optionally, a specific handle to close. The default value is '*'
-            which will attempt to close all open handles.
+            A specific handle to close.
         :type handle: str or ~azure.storage.file.Handle
-        :param bool recursive:
-            Boolean that specifies if operation should apply to the directory specified by the client,
-            its files, its subdirectories and their files. Default value is False.
         :keyword int timeout:
             The timeout parameter is expressed in seconds.
-        :returns: The number of file handles that were closed.
+        :returns:
+            The total number of handles closed. This could be 0 if the supplied
+            handle was not found.
         :rtype: int
         """
         timeout = kwargs.pop('timeout', None)
+        start_time = time.time()
         try:
             handle_id = handle.id # type: ignore
         except AttributeError:
-            handle_id = handle or '*'
-        command = functools.partial(
-            self._client.directory.force_close_handles,
-            handle_id,
-            timeout=timeout,
-            sharesnapshot=self.snapshot,
-            recursive=recursive,
-            cls=return_response_headers,
-            **kwargs)
-        try:
-            start_close = await command()
-        except StorageErrorException as error:
-            process_storage_error(error)
+            handle_id = handle
```

Comment:
Should we make sure handle_id is not '*' here...and may be force users to use close_all_handles if they want to delete everything

Reply:
Yup - will add.